backend/src/models/user.js

```diff
@@ -9,6 +9,6 @@ const User = new mongoose.Schema({
   email: String,
   key: String,
   role: String,
-  workspaceId: String
+  cycleId: mongoose.Schema.Types.ObjectId
```

Comment:
why is user bound to `cycleId` and not `workspaceId`?

Reply:
oh whoops i went through and changed all of them without realizing what User was- good catch